scss/tech.scss

```diff
@@ -49,3 +50,26 @@
   margin: 20px 0;
   text-align: center;
 }
+
+@media (max-width: 1024px) {
```

Comment:
I think all this is not needed in this PR. Else, add the same fixes than responsive PR